npt/util.py

```diff
@@ -596,5 +603,11 @@ def parse_cmdline() -> OptionContainer:
     return opt
 
 
+def read_usr_opts() -> OptionContainer :
+    args = sys.argv.copy() 
```

Comment:
Why does this ready `sys.argv`? It seems like it could take the arguments as a parameter, and so make npt/util.py completely independent on `sys.argv`. Would make it easier to test.

Reply:
I think it's far cleaner to make this file independent of `sys.argv`. It also makes it easier to test.

Reply:
Okay. will lift  sys.argv   to "__main__.py"

Reply:
Thanks!

Reply:
Done as well. You might want to review the location in main.py where I am passing in sys.argv into util.py . It is the least ugly hack that I could come up with.